Explaining how we work with Partners (#9135)
PostHog/posthog.com: contents/handbook/comms/partners.md

---
title: Partners
sidebar: Handbook
showTitle: true
---

> We're frequently contacted about revenue-sharing partnerships, or individuals and agencies that want to be listed as official partners. We never do this and do not operate any form of partnership program to help users implement, extend, or adapt PostHog to their needs.

Reviewer: ```suggestion
We're frequently contacted about revenue-sharing partnerships, or individuals and agencies that want to be listed as official partners. We do not currently operate any form of partnership program to help users implement, extend, or adapt PostHog to their needs.
```

**If someone contacts you about partnering with PostHog**, refer them to [Joe](https://posthog.com/community/profiles/29070). He'll give them the bad news and explore any opportunities.  

Instead, we recommend users who need this sort of help explore our existing resources, or [contact us](http://app.posthog.com/home#supportModal). If we can help, we will!

## Helpful resources for users
Users who contact us about wanting support from a partner often want particular types of help. We've curated some resources below which we can give them so they can self-serve where possible. 

### Migration help

> If a customer contacts us about migrating data into PostHog we should first refer them to the Sales and CS Team, who will advise on best practices, help the team avoid rate limits, and ensure they get appropriate support. We also have guides to help teams migrate data on their own.

Reviewer: ```suggestion
If a customer contacts us about migrating data into PostHog we should first refer them to the Sales & CS Team, who will triage them. We also have guides to help teams migrate data on their own.
```

- [Migrating from Amplitude to PostHog](/docs/migrate/migrate-from-amplitude)
- [Migrating from Mixpanel to PostHog](/docs/migrate/mixpanel)
- [Migrating from Heap to PostHog](/docs/migrate/heap)
- [Migrating from a self-hosted deployment to PostHog Cloud](/docs/migrate/migrate-to-cloud#between-cloud-instances-eg-us-cloud-to-eu-cloud)
- [Syncing other platforms to our data warehouse](/docs/data-warehouse/setup)

### Implementation help

> Sometimes teams want help or advice on their event taxonomy, or creating specific insights. Paying users should generally be referred to the Sales & CS team, who can advise on best practices. We also have [a wide variety of dashboard templates](/templates) and tutorials to help teams get started.

Reviewer: ```suggestion
Sometimes teams want help or advice on their event taxonomy, or creating specific insights. Users who look like they have the potential to pay >$20k should generally be referred to the Sales & CS team, otherwise they should go through the regular support flow. We also have [a wide variety of dashboard templates](/templates) and tutorials to help teams get started.
```
